release_notes/release-notes.md

```diff
@@ -0,0 +1,50 @@
+## Version 1.1.0
```

Comment:
This file will not get processed when uploading to splunkbase

There needs to be one release notes file per version in this directory with the version as the filename (e.g. `1.1.0.md`). Also, generally you do not need to manually manage that. Anything you put in `unreleased.md` will get appended to a file with the name matching the version in the app JSON when the pull request from next -> main is automatically generated (which happens when you merge a PR into the next branch).

If you want to retroactively add the release note history for prior versions, you should split this up into separate files for consistency anyways. Although, it will not update the release notes for past releases either way.

Reply:
Yeah, that works.